Basic/Pod/DeveloperGuide.pod

```diff
@@ -178,34 +140,39 @@ a PDL installation.  For some modules the .pm files are generated
-  | PDL::NiceSlice         | Basic/SourceFilter/NiceSlice.pm |
-  | PDL::Ops               | Basic/Ops/ops.pd                |
-  | PDL::Primitive         | Basic/Primitive/primitive.pd    |
-  | PDL::Slices            | Basic/Slices/slices.pd          |
-  | PDL::Ufunc             | Basic/Ufunc/ufunc.pd            |
-
-
+  | Package                                | Source file                     |
+  |----------------------------------------+---------------------------------|
+  | PDL                                    | Basic/PDL.pm                    |
+  | PDL::Bad                               | Basic/Bad/bad.pd                |
+  | PDL::Basic                             | Basic/Core/Basic.pm             |
+  | PDL::Core                              | Basic/Core/Core.pm              |
+  | PDL::Demos                             | Demos/Demos.pm                  |
+  | PDL::Graphics::Gnuplot                 | *not* in this repository        |
+  | PDL::Graphics::PGPLOT                  | *not* in this repository        |
+  | PDL::Graphics::PLplot                  | *not* in this repository        |
+  | PDL::Graphics::Prima                   | *not* in this repository        |
+  | PDL::Graphics::Simple                  | *not* in this repository        |
+  | PDL::Graphics::TriD                    | Graphics/TriD/TriD.pm           |
+  | PDL::Graphics::TriD::*                 | Graphics/TriD/Objects.pm        |
+  | PDL::Graphics::TriD::BoundingBox       | Graphics/TriD/TriD.pm           |
+  | PDL::Graphics::TriD::Material          | Graphics/TriD/TriD.pm           |
+  | PDL::Graphics::TriD::OneTransformation | Graphics/TriD/TriD.pm           |
+  | PDL::Graphics::TriD::QuaterController  | Graphics/TriD/TriD/ArcBall.pm   |
+  | PDL::Graphics::TriD::Scale             | Graphics/TriD/TriD.pm           |
+  | PDL::Graphics::TriD::Transformation    | Graphics/TriD/TriD.pm           |
+  | PDL::Graphics::TriD::Translation       | Graphics/TriD/TriD.pm           |
+  | PDL::IO::FITS                          | IO/FITS/FITS.pm                 |
+  | PDL::IO::Misc                          | IO/Misc/misc.pd                 |
+  | PDL::IO::Pic                           | IO/Pnm/pnm.pd                   |
+  | PDL::IO::Storable                      | IO/Storable/storable.pd         |
+  | PDL::Lvalue                            | Basic/Lvalue.pm                 |
+  | PDL::Math                              | Basic/Math/math.pd              |
+  | PDL::MatrixOps                         | Basic/MatrixOps/matrixops.pd    |
+  | PDL::NiceSlice                         | Basic/SourceFilter/NiceSlice.pm |
+  | PDL::Ops                               | Basic/Ops/ops.pd                |
+  | PDL::Primitive                         | Basic/Primitive/primitive.pd    |
+  | PDL::Slices                            | Basic/Slices/slices.pd          |
+  | PDL::Ufunc                             | Basic/Ufunc/ufunc.pd            |
```

Comment:
This section is a maintenance nightmare that might go out of date almost immediately. Can you remove the "not in this repo" ones, and keep it to only ones under `Basic`? Also, the whole thing can probably be replaced with advice to use `git grep PACKAGENAME`.

Reply:
I'll remove the "not in this repo ones".  However, I don't think that the information is very volatile. I would actually welcome changes which push the layout into a CPAN mainstream layout and volunteer to adapt the table accordingly.
This table enables me to directly jump to the source file of a package in my editor while `git grep PACKAGENAME` gives false positives.